storage/storagedriver/overlayfs-driver.md

```diff
@@ -11,26 +11,32 @@ and with a simpler implementation. Docker provides two storage drivers for
 OverlayFS: the original `overlay`, and the newer and more stable `overlay2`.
 
 This topic refers to the Linux kernel driver as `OverlayFS` and to the Docker
-storage driver as `overlay` or `overlay2`.
-
-> **Note**: If you use OverlayFS, use the `overlay2` driver rather than the
-> `overlay` driver, because it is more efficient in terms of inode utilization.
-> To use the new driver, you need version 4.0 or higher of the Linux kernel,
-> unless you are a Docker EE user on RHEL or CentOS, in which case you need
-> version 3.10.0-514 or higher of the kernel and to follow some extra steps.
->
+storage driver as `overlay2`.
+
+> **Note**: The `overlay` driver is deprecated in Docker for 18.09 and later.
+> Docker recommends `overlay2` for its storage driver.
+> 
```

Comment:
The information about what kernel is required for overlay2 ("you need version 4.0 or higher of the Linux kernel"...) is useful and should not be removed.